docs/getting-started/emojivoto-instrumented.yaml

```diff
@@ -56,7 +39,7 @@ spec:
             requests:
               cpu: 100m
         - name: emojivoto-emoji-instrumentation
-          image: keyval/otel-go-agent:v0.6.0
+          image: otel-go-instrumentation:v0.1
```

Comment:
How do you feel about setting the image as `otel-go-instrumentation` (without version suffix) and updating the demo prerequisites to require the docker image be created first with`make docker-build`?

Then we can update the image to an published one once #66 is done.

Reply:
Updated based on your feedback 👍